src/lib/containers/SynapseTable.js

```diff
@@ -254,13 +254,42 @@ export default class SynapseTable extends React.Component {
         // shouldn't be displayed
         let pastZero = this.props.getLastQueryRequest().query.offset > 0
 
+        let x_data = [];
+        data.facets.forEach(item => {
+            if (item.facetType === "enumeration" && item.columnName === this.props.filter) {
+                item.facetValues.forEach(facetValue => {
+                    if (item.columnName) {
+                        x_data.push({ columnName: item.columnName, ...facetValue });
+                    }
+                });
+            }
+        });
+        
+        // edge case -- if they are all false then they are considered all true..
+        // sum up the counts of data
+        let allFalse = false
+        let totalAllFalseCase = 0
+        let totalStandardCase = 0
+        for (let key in x_data) {
+             if (x_data.hasOwnProperty(key)) { 
+                 allFalse = allFalse || x_data[key].isSelected
+                 totalAllFalseCase += x_data[key].count 
+                 totalStandardCase += x_data[key].isSelected ?  x_data[key].count : 0
+            } 
+        }
+        let total = allFalse === false ? totalAllFalseCase: totalStandardCase
```

Comment:
This allFalse logic is bewildering.  Think we need a rename and simplification here.
allFalse => anyTrue?
Then 
`let total = allFalse === false ? totalAllFalseCase: totalStandardCase` would become this, right?
`let total = anyTrue ? totalStandardCase : totalAllFalseCase`